sites.json

```diff
@@ -31,6 +31,120 @@
             ]
         },
         {
+           "name": "Social Media",
+           "linkname": "socialmedia",
+           "sites": [
+              {
+                "name": "LinkedIn",
+                "domain": "www.linkedin.com"
+              },
+              {
+                "name": "Facebook",
+                "domain": "www.facebook.com"
+              },
+              {
+                "name": "Twitter",
+                "domain": "www.twitter.com"
+              },
+              {
+                "name": "Pintrest",
+                "domain": "www.pinterest.com"
+              },
+              {
+                "name": "Tumblr",
+                "domain": "www.tumblr.com"
+              },
+              {
+                "name": "Instagram",
+                "domain": "www.instagram.com"
+              },
+              {
+                "name": "Vine",
+                "domain": "www.vine.com"
+              },
+              {
+                "name": "Vimeo",
+                "domain": "www.vimeo.com"
+              },
+              {
+                "name": "Match.com",
+                "domain": "www.match.com"
+              },
+              {
+                "name": "OkCupid",
+                "domain": "www.okcupid.com"
+              }
+           ]
+        },
+        {
+            "name": "E-Commerce Sites",
```

Comment:
I suggest renaming to "Commerce". We know they're sites and I think the web is mature enough that we can start dropping the `e-` on things :wink: